pkg/util/validation/validation_test.go

```diff
@@ -554,3 +555,76 @@ func TestValidatePropagationSpec(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateApplicationFailover(t *testing.T) {
+	var correctSeconds int32 = 100
```

Comment:
Consider all Seconds are point type, we may still need these temporary variables.

Reply:
pointer.Int32(100)

Reply:
Thanks for reminding.